src/file-uploader/file-uploader.component.ts

```diff
@@ -147,7 +147,6 @@ export class FileUploader implements OnInit {
 		// overrides the undefined files value set by the user
 		if (!this.files) {
 			this.files = new Set();
-			this.filesChange.emit(this.files);
```

Comment:
Can `this.files` be initialized when declared on line 123 `@Input() files: Set<FileItem>;` as `@Input() files = new Set<FileItem>();` (or something similar)?
If everything still works as intended (it should), then remove `ngOnInit` completely.
I thought that was what we talked about.